src/main/java/com/minecolonies/coremod/sounds/DeliverymanSounds.java

```diff
@@ -1,5 +1,6 @@
 package com.minecolonies.coremod.sounds;
 
+import com.minecolonies.api.util.constant.Suppression;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.util.constant.Suppression'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)